client/src/components/icons/LogoutIcon.js

```diff
@@ -13,17 +15,22 @@ const Logout = styled.path`
   :hover {
     fill: ${props => props.theme.colors.secondary};
   }
+  :active {
+    fill: ${props => props.theme.colors.primary};
+  }
 `;
 
 export default function LogoutIcon() {
```

Comment:
if you think about changing the Icon in the future add props here in the () and ...